packages/Operators/src/Factory/DTK_PointCloudOperatorFactory.hpp

```diff
@@ -90,7 +90,8 @@ class PointCloudOperatorFactory
     enum PointCloudOperatorType
     {
 	SPLINE_INTERPOLATION,
-	MOVING_LEAST_SQUARE
+	MOVING_LEAST_SQUARE,
+        NODE_TO_NODE
     };
```

Comment:
also it might be a good idea to use a static map (``std::string`` to builder) and use register in derived classes.
the switch statement is hard to maintain and you have to recompile each time you add a new operator.